src/test/compile-fail/const-pattern-irrefutable.rs

```diff
@@ -9,8 +9,8 @@
 // except according to those terms.
 
 mod foo {
-    pub const b: u8 = 2; //~ NOTE constant defined here
-    pub const d: u8 = 2; //~ NOTE constant defined here
```

Comment:
I can re-implement the original behavior if desired, but I think it's better to be consistent and only emit one note.